app/src/main/java/org/coepi/android/ui/alerts/AlertsAdapter.kt

```diff
@@ -15,6 +15,7 @@ import org.coepi.android.ui.alerts.AlertCellViewData.Item
 
 class AlertsAdapter(
     private val onAlertClick: (AlertViewData) -> Unit,
+    private val onAlertDismissed: () -> Unit,
     private val alertsRepo: AlertsRepo
```

Comment:
This escaped review. The adapter shouldn't have a reference to the alerts repo. Call `alertsRepo.removeAlert(item.viewData.alert)` in the view model, as part of `onAlertDismissed()`